sylvia-derive/src/message.rs

```diff
@@ -1382,7 +1395,23 @@ impl<'a> GlueMessage<'a> {
                     Err(D::Error::custom(err_msg))
                 }
             }
+        };
+
+        // Append this derive for cw-orch contracts
+        // (Optionally you could always do this?)
+        #[cfg(feature = "orch")]
```

Comment:
Change #2: Add the From statement for the contract specific message.

(Note: we want to extend this for all the messages from all the interface variants, but that proved hard)